### PR TITLE
Feature/activities permissions

### PR DESCRIPTION
**Pull request by annashamray**
fixes https://github.com/GemeenteUtrecht/ZGW/issues/748

### CODE REVIEW

File: backend/src/zac/accounts/permission_loaders.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def add_atomic_permission_to_user(
     atomic_permission = AtomicPermission.objects.create(
         object_type=object_type,
         object_url=object_url,
-        permission=zaken_inzien.name,
+        permission=permission_name,
```
Comment (damm89) [inline backend/src/zac/accounts/permission_loaders.py:35]: good catch(?)

File: backend/src/zac/activities/api/permission_loaders.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+from ..models import Activity
+from ..permissions import activiteiten_schrijven, activities_read
+
+
+def add_permissions_for_activity_assignee(activity: Activity) -> List[AtomicPermission]:
+    atomic_permissions = []
+    for permission in [activities_read, activiteiten_schrijven]:
+        atomic_permission = add_atomic_permission_to_user(
+            activity.assignee, activity.zaak, permission_name=permission.name
```
Comment (damm89) [inline backend/src/zac/activities/api/permission_loaders.py:14]: Activity.zaak shouldn't be activity.url? In the current implementation the atomic permission relates to the zaak and not the activity.

File: backend/src/zac/api/permissions.py [hunk 3]
```diff
@@ -79,6 +81,9 @@ class ObjectDefinitionBasePermission(DefinitionBasePermission):
     def get_object(self, request: Request, obj_url: str):
         raise NotImplementedError("This method must be implemented by a subclass")
 
+    def get_object_url(self, serializer=None) -> str:
+        return serializer.validated_data[self.object_attr]
```
Comment (damm89) [inline backend/src/zac/api/permissions.py:85]: if serializer = None this will break

**Review by damm89 — COMMENTED** [review]
Looks, good. 1 remark and 1 question

File: backend/src/zac/activities/api/permission_loaders.py (hunk 2 above)
Comment (annashamray) [reply backend/src/zac/activities/api/permission_loaders.py:14]: yeah, `object_type` for atomic permissions (and also blueprint permissions) can be `zaak` or `document` now, and we grant all permissions per zaak or per document

File: backend/src/zac/api/permissions.py (hunk 3 above)
Comment (annashamray) [reply backend/src/zac/api/permissions.py:85]: Nice catch! I'll remove None as default value

**Review by damm89 — APPROVED** [review]

**Pull request merged**